CONTRIBUTING.md

```diff
@@ -57,3 +57,15 @@ Ensure to remove/update generated files under `src` if they cause problems.
     ```
 
 Ensure to remove/update generated files under `src` if they cause problems.
+
+## Interactively run sbt scripted tests
+
+There are a number of Scala projects being tested as 'scripted' tests. After a
+scripted test has completed (successfully or not) it removes the temporary
+directory where it was running.
+
+To interactively inspect the project, you can insert a `$ pause` command in the
```

Comment:
I tried
```
> compile
$ pause
```
but that will of course not work, since `compile` fails first.

Can we add clarification that it can be scripted like:
```
> protocGenerate
> generateUnmanaged
$ pause
```
or whatever you recommend instead